lib/src/components/form/form.dart

```diff
@@ -1444,6 +1444,7 @@ class SubmitButton extends StatefulWidget {
 class _SubmitButtonState extends widgets.State<SubmitButton> {
   Future<bool>? _future;
   FormController? _controller;
+  bool _isSubmitting = false;
```

Comment:
Can you elaborate on why we need this field here? The FutureBuilder should handle this one already, or am I missing something?

Reply:
I don't think the issue is caused by the submit button, checked it myself yesterday, it was the unnecessary microtask inside the Form component. I pushed a commit that solves this issue, you can rebase it and this change is no longer necessary.

Reply:
Okay. Thank you